content/en/platform/corda/5.0-alpha/deploying/group-policy.md

```diff
@@ -0,0 +1,195 @@
+
+## Group Policy Types
+
+You can create the following types of group policy files:
+* [MGM Group Policy](#mgm-group-policy) — defines the information for an [MGM](../introduction/key-concepts.html#membership-management) of a [dynamic network](../deploying/network-types.html#dynamic-networks).
+* [Dynamic Network Member Group Policy](#dynamic-network-member-group-policy) — defines the information for members in a [dynamic network](network-types.html#dynamic-networks).
+* [Static Network Member Group Policy](#static-network-member-group-policy) — defines the information for members in a [static network](network-types.html#static-networks).
+
+### MGM Group Policy
+
+You must define a group policy file for an MGM. This file is included in the CPI that creates the virtual node of the MGM.
+As most of the information in a group policy file is exported by the MGM, the initial MGM group policy is a much smaller file than that needed to create a member.
+
+During onboarding, the MGM receives all of the information defining the group.
+From that information, the MGM builds and exports a group policy file for members to use.
+
+The MGM group policy file only requires a flag to indicate that a group ID must be generated during virtual node onboarding and information about how to register itself as part of the group.
+Registration for an MGM is essentially finalising setup of the the group, but the registration terminology is kept in-line with the member setup.
+
+This is a simple file that you can construct manually. You can also use the Corda CLI to export a default version.
+
+For example:
+
+``` json
+{
+  "fileFormatVersion": 1,
+  "groupId": "CREATE_ID",
+  "registrationProtocol": "net.corda.membership.impl.registration.dynamic.mgm.MGMRegistrationService",
+  "synchronisationProtocol": "net.corda.membership.impl.synchronisation.MgmSynchronisationServiceImpl"
+}
+```
+
+Alternatively, with optional parameters:
+``` json
+{
+  "fileFormatVersion": 1,
+  "groupId": "CREATE_ID",
+  "registrationProtocol": "net.corda.membership.impl.registration.dynamic.mgm.MGMRegistrationService",
+  "synchronisationProtocol": "net.corda.membership.impl.synchronisation.MgmSynchronisationServiceImpl",
+  "protocolParameters": {
+    "foo": "bar"
+  }
+}
+```
+
+For more information, see [MGM Onboarding](../deployment-tutorials/membership/mgm-onboarding.html).
+
+### Dynamic Network Member Group Policy
+
+To join a group, a member must use a group policy file that was exported from the MGM of that group.
+This group policy file contains much more information than the initial file used by the MGM.
+It contains the group ID, the protocol to join that group, the protocol to use to sync data with the MGM, and any additional parameters that are needed for these protocols.
+It also contains the full `MemberInfo` of the MGM so that it can be populated in the member's member list cache to allow communication with the MGM during registration.
```

Comment:
```suggestion
It also contains the full `MemberInfo` of the MGM, so that it can be populated in the member's member list cache to allow communication with the MGM during registration.
```